.github/workflows/master.yml

```diff
@@ -58,7 +58,17 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: reuse lint
-        run: make check-reuse
+        uses: fsfe/reuse-action@v2
+
+  fossa-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: FOSSA scan
+        uses: fossa-contrib/fossa-action@v3
+        with:
+          fossa-api-key: 9dc8fa92e3dd565687317beb87b56d89
```

Comment:
Maybe this would be better in a GitHub secret? https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions

Reply:
Hi @ghislainbourgeois, as per the FOSSA GitHub Action, having it as a secret does not work. Please see here the usage: https://github.com/marketplace/actions/fossa-action#usage